tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = py26,py27,lint,docs
+skipsdist = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}/readthedocs:{toxinidir}
+    DJANGO_SETTINGS_MODULE=settings.test
+deps = -r{toxinidir}/requirements/pip.txt
```

Comment:
I think, based on [depending on requitements.txt](https://testrun.org/tox/latest/example/basic.html#depending-on-requirements-txt) that all commands are already executed with `{toxinidir}` as the working directory, making the inclusion of it in your paths redundant.